frontend/src/components/CodeCheck.tsx

```
@@ -0,0 +1,183 @@
+    submitTestCase,
+    submitRuntimeResponse,
+} from "@/api/api";
+import RuntimeCheck from "./RuntimeCheck";
+
+interface CodeCheckProps {
+    moduleId: number;
+    onTestCasesCompleted: (completed: boolean) => void;
+}
+
+// TODO: use this for all?
+interface TestCase {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'TestCase'. Consider to delete it or rename it.

'TestCase' is defined here:


(lint/suspicious/noRedeclare)